python-sdk/src/astro/sql/operators/append.py

```diff
@@ -59,8 +62,43 @@ def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
             target_table=self.target_table,
             source_to_target_columns_map=self.columns,
         )
+
         return self.target_table
 
+    def get_openlineage_facets(self) -> OpenLineageFacets:
+        input_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=self.source_table.openlineage_dataset_namespace(),
+                name=self.source_table.openlineage_dataset_name(),
+                facets={
+                    "table_name": self.source_table.name,
+                    "row_affected": 0,  # FixMe
```

Comment:
Is this always 0?

Reply:
look like our Table object does not have the row count data. do you think it make sense to expose the `get_row_count` on Table object for some database query like `select count(*) from xyz` time taking query

Reply:
Yes I was thinking something like following in BaseTable class:
```
    def get_rows(self):
        db_connection = create_engine(self.conn_id)
        db_metadata = MetaData(bind=db_connection, reflect=True)
        sqlalchemy_table = db_metadata.tables[self.name]
        with Session(db_connection) as session:
            return session.query(sqlalchemy_table).count()
```

Reply:
expose row count on BaseTable